Sources/DeepLearning/Operators/Comparison.swift

```diff
@@ -0,0 +1,232 @@
+    @inlinable @inline(__always)
+    static func .>= (lhs: Tensor, rhs: Scalar) -> Tensor<Bool> {
+        return Raw.greaterEqual(lhs, Tensor(rhs))
+    }
+}
+
+extension Tensor: Comparable where Scalar: Numeric & Comparable {
+    /// Returns a Boolean value indicating whether the value of the first argument is
+    /// lexicographically less than that of the second argument.
+    @inlinable @inline(__always)
+    public static func < (lhs: Tensor, rhs: Tensor) -> Bool {
+        return (lhs .< rhs).all()
+    }
+
+    /// Returns a Boolean value indicating whether the value of the first argument is
+    /// lexicographically less than or equal to that of the second argument.
+    @inlinable @inline(__always)
+    public static func <= (lhs: Tensor, rhs: Tensor) -> Bool {
+        return (lhs .<= rhs).all()
+    }
+
+    /// Returns a Boolean value indicating whether the value of the first argument is
+    /// lexicographically greater than that of the second argument.
+    @inlinable @inline(__always)
+    public static func > (lhs: Tensor, rhs: Tensor) -> Bool {
+        return (lhs .> rhs).all()
+    }
+
+    /// Returns a Boolean value indicating whether the value of the first argument is
+    /// lexicographically greater than or equal to that of the second argument.
+    @inlinable @inline(__always)
+    public static func >= (lhs: Tensor, rhs: Tensor) -> Bool {
+        return (lhs .>= rhs).all()
+    }
+}
+
+public extension Tensor where Scalar: Numeric & Comparable {
+    /// Returns a Boolean value indicating whether the value of the first argument is
+    /// lexicographically less than that of the second argument.
+    @inlinable @inline(__always)
+    static func < (lhs: Tensor, rhs: Scalar) -> Bool {
+        return (lhs .< rhs).all()
+    }
+
+    /// Returns a Boolean value indicating whether the value of the first argument is
+    /// lexicographically less than or equal to that of the second argument.
+    @inlinable @inline(__always)
+    static func <= (lhs: Tensor, rhs: Scalar) -> Bool {
+        return (lhs .<= rhs).all()
+    }
+
+    /// Returns a Boolean value indicating whether the value of the first argument is
+    /// lexicographically greater than that of the second argument.
+    @inlinable @inline(__always)
+    static func > (lhs: Tensor, rhs: Scalar) -> Bool {
+        return (lhs .> rhs).all()
+    }
+
+    /// Returns a Boolean value indicating whether the value of the first argument is
+    /// lexicographically greater than or equal to that of the second argument.
+    @inlinable @inline(__always)
+    static func >= (lhs: Tensor, rhs: Scalar) -> Bool {
+        return (lhs .>= rhs).all()
+    }
+}
+
+public extension Tensor where Scalar: Equatable {
+    /// Computes `lhs != rhs` element-wise and returns a `Tensor` of Boolean scalars.
+    /// - Note: `.==` supports broadcasting.
+    @inlinable @inline(__always)
+    static func .==(lhs: Tensor, rhs: Tensor) -> Tensor<Bool> {
```

Comment:
```suggestion
    static func .== (lhs: Tensor, rhs: Tensor) -> Tensor<Bool> {
```
Could you make it so that every operator function has a space between the operator and the left parenthesis?